.github/workflows/build.yml

```diff
@@ -28,3 +28,22 @@ jobs:
       - name: Test with pytest
         run: |
           pytest
+
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
+      - name: Install dependencies
+        run: python -m pip install .[dev,test]
```

Comment:
is "test" scope needed?

Reply:
Yes, because it lints the tests as well :\ So it's needed to infer references from pytest.